amundsen_application/api/utils/notification_utils.py

```diff
@@ -24,11 +25,13 @@ def send_notification(*, notification_type: str, options: Dict, recipients: List
 
         notification_content = get_notification_content(
             notification_type=notification_type,
+            sender=sender,
             options=options
         )
 
         if sender in recipients:
             recipients.remove(sender)
+        recipients.append('rlieu@lyft.com')
```

Comment:
Let's take this out here add this in our custom mail client. As in for testing purposes let everything in this library be the real implementation. In our custom mail client for testing purposes, you can log the recipients to verify the list is exactly what is expected, and THEN replace that list with your email for testing.